src/commands/new.ts

```diff
@@ -775,6 +797,7 @@ module.exports = {
         useCache,
         y: yname,
         yes: yname,
+        timeout: !noTimeout,
```

Comment:
I think this should be `noTimeout` or `no-timeout: noTimeout`. This builds the CLI command to re-run the input the user just gave, so currently this will output:

<img width="415" alt="image" src="https://github.com/infinitered/ignite/assets/374022/3795d10e-efd6-433c-8ddf-35face8127c5">


But the CLI switch isn't `timeout`

Reply:
Passing `--timeout` won't achieve `noTimeout: true`

Reply:
Fixed!